src/parser.ts

```diff
@@ -40,9 +40,6 @@ export const getParser = (parser: Parser["parse"]) =>
 
     /**
      * Control order of tags by weights. Smaller value brings tag higher.
-     *
-     * @param {String} tagTitle TODO
-     * @returns {Number} Tag weight
```

Comment:
Keep returns tag with desscription

Reply:
The function is called `TagOrderWeight`. This return description doesn't contain any new information. That's why I removed it.

Reply:
OK.